src/data/reducers/courseInfo.js

```diff
@@ -0,0 +1,105 @@
+import DiscoveryDataApiService from '../services/DiscoveryDataApiService';
+
+
+// *** CONSTANTS ***
+
+export const FAIL_COURSE_INFO = 'FAIL_COURSE_INFO';
+export const RECEIVE_COURSE_INFO = 'RECEIVE_COURSE_INFO';
+export const REQUEST_COURSE_INFO = 'REQUEST_COURSE_INFO';
```

Comment:
@AlasdairSwan may have already commented on this, but do we want a Constants file at some point?